psique/src/App/test/wisc/subtest/matrices.js

```diff
@@ -1,8 +1,311 @@
+  function imagenInit(item){
+    
+    if(item!==1){
+      let arrayAux = results
+      for (let i = 0; i < item-1; i++) {
+        arrayAux[i]=1
+      }
+      setResults(arrayAux)
+    }
+    else{
+      retornoHecho=false
+    }
+
+    firstItem=item
+    setNumberItem(item)
+    setState('ejemplo')
+  }
+
+  //Función que cambia el estado principal
+  function next(){
+    switch (state) {
+      case 'ejemplo':
+        setState('test')
+        break;
+      case 'results':
+        setState('revision')
+        setResultsAux(results)
+        break;
+      case 'instruccion':
+        setState('seleccion')
+        break;
+      case 'revision':
+        setState('results')
+        break;
+      default:
+        break;
+    }
+  }
+
+  function getResult() {
+    var total = 0;
+    for(var i=0;i<results.length;i++){
+      total = total + results[i];
+    }
+    return total;
+  }
+
+  function nextExample(){
+    switch(stateExample){
+      case 'ejemplo1':
+        setStateExample('ejemplo2') 
+        break;
+      case 'ejemplo2':
+        setStateExample('ejemplo3')
+        break;
+      case 'ejemplo3':
+        setState('test')
+        break;
+      default:
+        break;
+    }
+
+  }
+
+  function content(){
+    switch (state) {
+      case 'instruccion':
+        return (
+        <div id= "inicio" >
+        <h1>Matrices</h1>
+        <b>Intrucciones generales:</b>
+        <p>Se presentarán una matriz o una serie incompleta al paciente</p>
+        <p>La tarea es escoger, entre cinco opciones, la que mejor complete la matriz o la serie</p>
+        <br/>
+        <b>Intrucciones para registrar la respuesta de paciente:</b>
+        <br/>
+        <br/>
+        <p>Para calificar se debe presionar en el teclado el número correspondiente a la respuesta dada por el paciente </p>
+        <p>El sistema calificara automaticamente la prueba</p>
+        <br/>
+        <CustomButton
+          msj="Iniciar subprueba"
+          callback={next}
+        ></CustomButton>  
+      </div>)
+      
+      case "seleccion":
+        return(
+         <div >
+         <h1>Figuras incompletas</h1>
+         <p>En que estimulo desea iniciar la prueba? </p>
+         <p>Pacientes de edad 6-8 o con sospechas de discapacidad intelectual:</p>
+         <CustomButton msj="Estimulo 4"
+         callback={()=>imagenInit(4)}></CustomButton> 
+         <p>Pacientes de edad 9-11</p>
+         <CustomButton msj="Estimulo 7"
+         callback={()=>imagenInit(7)}></CustomButton> 
+         <p>Pacientes de edad 12-16</p>
+         <CustomButton msj="Estimulo 11"
+         callback={()=>imagenInit(11)}></CustomButton> 
+       </div>
+        )
+      case "ejemplo":
+        return(
+          <div >
+              <h1> Estimulo {state}</h1>
+              <img 
+              className={classes.img} 
+              alt={"Estimulo "+state} 
+              src={require("../../../assets/estimulos/matrices-wisc/"+stateExample+".jpg")} />
```

Comment:
The Extention .jpg is in uppercase in the filename, so the test throws an error to show the image.